torch/_decomp/decompositions.py

```diff
@@ -153,6 +153,12 @@ def hardsigmoid_backward(grad_output: Tensor, self: Tensor):
     )
 
 
+@register_decomposition(aten.hardtanh_)
```

Comment:
is this the right way to do it?

Reply:
I wouldn't land it this way into master but it's fine for branch

Reply:
for my education, what should be the proper way to implement this?

Reply:
We need a consolidated strategy for inplace decomps